api/README.md

```diff
@@ -90,6 +102,23 @@ Status: 200 OK
 ]
 ```
 
+## Get all categories
+```
+GET /categories
+```
+
+### Response
+Status: 200 OK
+```JSON
+[
+  {
+    "id": 1,
+    "category": "4x",
+    "identifierID": "85OKv8p5Ow"
```

Comment:
What is identifierID and how is that different from id? The former kind of sounds like identifierIdentifier

Reply:
This comes from the Board Game Atlas API (and is also in the database)